plugins/auth-keycloak/.amplicationrc.json

```
@@ -0,0 +1,9 @@
+{
+  "settings": {
+    "KEYCLOAK_HOST": "localhost",
+    "KEYCLOAK_REALM": "test-realm",
+    "KEYCLOAK_CLIENT_ID": "test-client",
+    "KEYCLOAK_CLIENT_SECRET": "pryUAZxDplBlJmRfCZ8yxH2NT1Dcad5H",
```

I would suggest not adding any secrets as default settings here. 
```suggestion
    "KEYCLOAK_CLIENT_SECRET": "DEFAULT_SECRET",
```